lib/utils.js

```diff
@@ -12,9 +12,9 @@ function linkFor (text) {
     .toLowerCase()
     .trim()
     .replace(/\s/g, '-')
-    .replace(/<(.*?)>(.+)<\/\1>/g, '$2') // e.g. exract 'a' from <kbd>a<kbd>
+    .replace(/(&nbsp;|<([^>]+)>)/ig, '') // this strips all <tags> from the title and just leaves the text in place.
```

Comment:
More than just the `<kbd>` tag, this applies to all tags that are nested in the title. Also, I'm not sure that the current regex actually does what it is intended to do because the `kbd` still appears in the title link whereas this fix removes that entirely. This means you have a nice clean title link such as `#nice-title` versus `#nice-kbdtitlekbd`.